### PR TITLE
add missing subcategories to aftershock recipes

### PR DESCRIPTION
**Pull request by dranitski**
Since recipe subcategories now mandatory, this pr fixes some crashes caused by Aftershock missing subcategories https://github.com/CleverRaven/Cataclysm-DDA/pull/51744#issuecomment-926499739

#### Summary
Bugfixes "Add missing subcategories to Aftershock recipes"

### CODE REVIEW

**Comment by svenstaro** [comment]
I was basically about to submit this very patch myself. :+1:

**Comment by jbytheway** [comment]
Hrm.  The CI on #51744 should have caught this but apparently didn't.  That's worrying...

**Comment by jbytheway** [comment]
Yeah, turns out that part of the CI was broken.  See #51843.

**Comment by gunblast** [comment]
Sweatbread should be in meat category, shouldn't it? It's organ meat not actual bread

**Comment by svenstaro** [comment]
Actually meat seems correct. Turns out I had no idea what [sweetbread](https://en.wikipedia.org/wiki/Sweetbread) is.

File: data/mods/Aftershock/recipes/recipe_overrides.json [hunk 1]
```diff
@@ -203,6 +207,7 @@
     "activity_level": "LIGHT_EXERCISE",
     "result": "grilled_sweetbread",
     "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREAD",
```
Comment (ZhilkinSerg) [inline data/mods/Aftershock/recipes/recipe_overrides.json:210]:
```suggestion
    "subcategory": "CSC_FOOD_MEAT",
```

**Review by Maleclypse — APPROVED** [review]

**Pull request merged**